app/pb_kits/playbook/pb_form/form_builder/checkbox_field.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Playbook
+  module PbForm
+    module FormBuilder
+      module CheckboxField
+        def check_box(name, data: {}, props: {})
+          value = props[:value] ||= 1
+          input = super(name, { required: props[:required], data: data }, value)
+          label_text = name.to_s.gsub("_", " ")
```

Comment:
We use Rails provided label method for this. That provides i18n lookup, and provides a uniform way of generating them label text.
https://github.com/powerhome/playbook/blob/3155894e73deeede3074a735f4bb307e2e9abe92/app/pb_kits/playbook/pb_form/form_builder/form_field_builder.rb#L9
```suggestion
          props[:label] = @template.label(@object_name, name) if props[:label] == true 
```

Reply:
Think I remember having difficulty implementing that `@template.label` but I may well be confusing that with that date picker form field I've been working on.  I'll give this another shot.